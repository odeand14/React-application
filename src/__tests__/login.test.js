const express = require("express");
const request = require("supertest");
const app = express();

const api = require("../App");
app.use(api);

describe("/login", () => {
    it("Should return a ")
});


const initServer = require("../../server/server");