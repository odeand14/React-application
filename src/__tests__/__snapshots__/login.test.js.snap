// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="container"
>
  <div
    className="text-center my-3"
  >
    <h1>
      Welcome to da monkeypage!
    </h1>
  </div>
  <hr />
  <div
    className="row"
  >
    <div
      className="justify-content-center col-4 mx-3 my-5"
    >
      <form
        acceptCharset="UTF-8"
        onSubmit={[Function]}
      >
        <h3
          className="my-3"
        >
          New user?
        </h3>
        <div
          className="form-group"
        >
          <label
            className="form-control-label"
          >
            Name:
          </label>
          <input
            className="form-control"
            placeholder="Name"
            required={true}
            type="text"
          />
          <small
            className="form-text text-muted"
          >
            Example: Firstname Lastname
          </small>
        </div>
        <div
          className="form-group"
        >
          <label
            className="form-control-label"
            htmlFor="newEmail"
          >
            Email:
          </label>
          <input
            className="form-control form-control-danger"
            id="newEmail"
            placeholder="e-mail"
            required={true}
            type="email"
          />
          <small
            className="form-text text-muted"
          >
            Example: youremail@something.com
          </small>
        </div>
        <div
          className="form-group"
        >
          <label
            className="form-control-label"
          >
            Password:
          </label>
          <input
            className="form-control"
            id="newPass"
            placeholder="Password"
            required={true}
            type="password"
          />
          <small
            className="form-text text-muted"
          >
            Example: No examples here! Youre on your own ;)
          </small>
        </div>
        <div
          className="form-group"
        >
          <label
            className="form-control-label"
          >
            Repeat password:
          </label>
          <input
            className="form-control"
            id="repeatPass"
            placeholder="Repeat password"
            required={true}
            type="password"
          />
          <small
            className="form-text text-muted"
          >
            Same input as above
          </small>
        </div>
        <div
          className="form-group text-center"
        >
          <button
            className="btn btn-primary mx-1 my-2 text-center"
            type="submit"
          >
            Create
          </button>
        </div>
      </form>
    </div>
    <div
      className="justify-content-center col-4 mx-3 my-5"
    >
      <form
        acceptCharset="UTF-8"
        className=""
        onSubmit={[Function]}
      >
        <h3
          className="my-3"
        >
          Existing user?
        </h3>
        <div
          className="form-group"
        >
          <label
            className="form-control-label"
            htmlFor="exampleInputEmail2"
          >
            Email:
          </label>
          <input
            className="form-control"
            id="oldEmail"
            placeholder="e-mail"
            required={true}
            type="email"
          />
          <small
            className="form-text text-muted"
          >
            Example: youremail@something.com
          </small>
        </div>
        <div
          className="form-group"
        >
          <label
            className="form-control-label"
            htmlFor="exampleInputPassword2"
          >
            Password:
          </label>
          <input
            className="form-control"
            id="oldPass"
            placeholder="password"
            required={true}
            type="password"
          />
          <small
            className="form-text text-muted"
          >
            Example: No examples here! Youre on your own ;)
          </small>
        </div>
        <div
          className="form-group text-center"
        >
          <button
            className="btn btn-info mx-1 my-2 text-center"
            type="submit"
          >
            Sign in
          </button>
        </div>
      </form>
    </div>
  </div>
  <hr />
</div>
`;
